Fluid.MvcViewEngine/FluidRendering.cs

```diff
@@ -62,6 +62,9 @@ static FluidRendering()
             context.AmbientValues[ViewPath] = path;
             context.AmbientValues.Add("Sections", new Dictionary<string, IList<Statement>>());
 
+            var viewsPath = Path.Combine(_hostingEnvironment.ContentRootPath, "Views");
```

Comment:
After deep thinking the mock `FileProvider` that I made in the test project is fine for the test, so I introduce a new `FileProviderWrapper` in the view engine only